### DISCUSSION TITLE
Large Figure with Long Caption (caption extends beyond bottom of page)

### DISCUSSION THREAD

**Discussion by emfeltham** [quarto-dev/quarto-cli#5553] [Q&A]
I am including figures into a .qmd document, via ```![this is a long caption](path/figure.svg)``` and exporting to a pdf via latex. My figures are getting cut off, and extending down into below the bottom of the page.

Is there any way to have the figure captions instead appear on the next page?

(I have searched around and found few to no solutions in the Tex community online. Some responses suggest that individuals shorten their captions or have a smaller figure -- this is not really an option for many situations where, say, a journal will want a long caption that explains figure details and the figure is detailed or has multiple panels.)

**Comment by mcanouil** [comment]
Could you share a small "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by jonas-hag** [reply]
Thank you for your work on quarto! Here is a reproducible example:

````qmd
---
title: "Reproducible Quarto Document"
format:
  pdf:
    documentclass: scrreport
---

```{r}
#| echo: false
#| fig-cap: "This showcases that long captions can be problematic. I have searched around and found few to no solutions in the Tex community online. Some responses suggest that individuals shorten their captions or have a smaller figure -- this is not really an option for many situations where, say, a journal will want a long caption that explains figure details and the figure is detailed or has multiple panels. I have searched around and found few to no solutions in the Tex community online. Some responses suggest that individuals shorten their captions or have a smaller figure -- this is not really an option for many situations where, say, a journal will want a long caption that explains figure details and the figure is detailed or has multiple panels. I have searched around and found few to no solutions in the Tex community online. Some responses suggest that individuals shorten their captions or have a smaller figure -- this is not really an option for many situations where, say, a journal will want a long caption that explains figure details and the figure is detailed or has multiple panels."
#| fig-width: 8
#| fig-height: 10
plot(cars)
```
````

**Reply by mcanouil** [reply]
That's LaTeX float system.
Quarto has no control on how it works. You need to know and use LaTeX.

**Reply by cscheid** [reply]
To expand on this:

> a journal will want a long caption that explains figure details

If a journal wants such deep control over your captions, they will also likely offer you a particular format to submit. In that case, you'd do one of the following

- use a Quarto format for that journal (in which case the journal is ok with the result, by definition)
- submit the `.tex` sources that Quarto outputs with `keep-tex: true`, in which case journal editors will do the final typesetting (and you won't have to worry about it, by definitions)

In our experience, what you're suggesting doesn't really come up often.

**Reply by jonas-hag** [reply]
Thank you for your explanations. Unfortunately it's for my thesis in a field where long captions are usual. I'll see if I can make a latex solution work

**Reply by cscheid** [reply]
(I was an academic prof before joining Quarto)

You should inquire with your university library about formatting, templates, and guidance. They should be able to at least point you in the right direction.

**Reply by jonas-hag** [reply]
> You should inquire with your university library about formatting, templates, and guidance. They should be able to at least point you in the right direction.

Thanks, I'll see if I find some guidance.

In the meantime if others have a similar problem, I found the approach to [have the complete caption on a new page](https://tex.stackexchange.com/questions/612618/figure-caption-on-the-next-page) the easiest to use. In contrast to what is shown in the link, I removed `\addcontentsline{lof}` that adds the short caption for the lof and added it directly in the `\caption` command in the `figure` environment because otherwise the style in the lof is not the same. However, the lof then links to the caption and not the actual figure. Another caveat is that you need to use `\ref{fig:my-label}` and can't use the quarto crossreference.

**Reply by mcanouil** [reply]
Note that you can use `fig-pos` option to set some LaTeX float position flag and you can use the pagebreak shortcode `{{< pagebreak >}}`.
